language-support/hs/bindings/examples/nim/package.yaml

```diff
@@ -0,0 +1,37 @@
+name:                nim
+version:             0.1.0.0
+description:         Play Nim on a DAML Ledger
+license:             Apache-2.0
+author:              The DAML Authors
+maintainer:          nick.chapman@digitalasset.com
+github:              https://github.com/digital-asset/daml
+
+dependencies:
+- ansi-terminal
+- base >= 4.7 && < 5
+- containers
+- daml-ledger
+- extra
+- haskeline
+- random
+- text
+- transformers
+- uuid
+
+executables:
+  nim-exe:
+    main:                DA.Ledger.App.Nim.Main
+    source-dirs:         src
+    ghc-options:
+    - -Wall
+    - -Wno-name-shadowing
+    - -threaded
+    - -rtsopts
+    - -with-rtsopts=-N
```

Comment:
Not really important here since we do not care about performance for this example (at least for now) but this doesn’t quite match the RTS options that we use via bazel: `    "-with-rtsopts=-N2 -qg -I0",`

Reply:
I tried this, but it broke the `stack build`:
```
ghc: unrecognised flag: -qg
did you mean one of:
  -g
```
@moritzkiefer-da Do you understand why this works in the `bazel` build?

Reply:
Looks like a quoting issue where stack passes `-qg` as a separate argument. Did you include the quotes I had there?

Reply:
I did.

Reply:
How does the generated cabal file look like? Maybe you need two layers of quoting, one for yaml and then one that ends up in the cabal file or something silly like that.

Reply:
See https://github.com/sol/hpack/issues/237

Reply:
This did the trick! (single quotes around double quotes)
```
    - '"-with-rtsopts=-N2 -qg -I0"'
```

Reply:
yaml is great …